azure-pipelines.yml

```diff
@@ -146,7 +146,6 @@ stages:
 
```

Comment:
I think you might need to do another rebase making sure your remotes are up to date.

Reply:
Yeah thinking of the same! Thanks!

Reply:
Have you made any progress on this?

Reply:
I am trying...let's see if this works!! else again will try. I am so sorry about the inconvenience.

Reply:
It is fine. Git has a complex learning curve.

Reply:
Thanks Nabil! I am kinda in my learning process! I will figure it out soon i believe